apps/rss_feeds/models.py

```diff
@@ -1086,6 +1087,12 @@ def calculate_scores(cls, facet):
     
     @property
     def user_agent(self):
+        feed_parts = urlparse.urlparse(self.feed_address)
```

Comment:
It would probably be fine to skip `urlparse` and just do a `.find` directly on `self.feed_address`, but I was trying to be as specific as possible.